ui/src/shared/components/ConfirmButtons.js

```diff
@@ -15,20 +15,23 @@ const ConfirmButtons = ({onConfirm, item, onCancel, buttonSize}) => (
       className={classnames('btn btn-success btn-square', {
         [buttonSize]: buttonSize,
       })}
+      disabled={!isSavable}
```

Comment:
This is a good idea, but the bool (and its semantics) will need to be flipped, because anywhere else that `<ConfirmButtons>` are used in the codebase will by default be disabled. The default should be enabled so that others don't break, and we don't have to pass in anything to get an enabled button.